templates/ai/index.html

```diff
@@ -0,0 +1,688 @@
+    <div class="p-divider row">
+      <div class="col-4 p-divider__block">
+        <div class="p-heading-icon--muted">
+          <div class="p-heading-icon__header">
+            {{ image (
+              url="https://assets.ubuntu.com/v1/6e184942-Webinar.svg",
+              alt="Webinar icon",
+              width="32",
+              height="28",
+              hi_def=True,
+              loading="auto|lazy",
+              attrs={"class":"p-heading-icon__img"}
+              ) | safe
+            }}
+            <h4 class="p-heading-icon__title">Webinar</h4>
+          </div>
+          <ul>
+            <li><a href="https://www.brighttalk.com/webcast/6793/427925">AI to production&nbsp;&rsaquo;</a></li>
+            <li><a href="https://www.brighttalk.com/webcast/6793/424551/demystifying-kubeflow-pipelines-from-tensorflow-notebook-to-pipeline">Kubeflow Pipelines&nbsp;&rsaquo;</a></li>
+            <li><a
+              href="https://www.brighttalk.com/webcast/6793/451777??utm_source=marketo&utm_medium=email&mkt_tok=eyJpIjoiTnpKaU9EazROR1F6WXpndyIsInQiOiI4NVBCRldTbG8zRTVrajg4eENSOEpaeGhwaWN4WFNmOEJVdlZiMERHNDJ6TWFKSkw3VE5WMWlMeGo5VmFUNVA0enFIT0lHVFduYzhFOGwwNGdUMFNTcUQ0aG9neDVYeHdndVpMRWVMQjFrRWdaVlwvZVA5anVYU2krYmdUbURJSEUifQ%3D%3D">AI deployment and inference&nbsp;&rsaquo;</a></li>
+          </ul>
+        </div>
+      </div>
+      <div class="col-4 p-divider__block">
+        <div class="p-heading-icon--muted">
+          <div class="p-heading-icon__header">
+            {{ image (
+              url="https://assets.ubuntu.com/v1/df54bb6a-document-open-icon.svg",
+              alt="Document icon",
+              width="32",
+              height="28",
+              hi_def=True,
+              loading="auto|lazy",
+              attrs={"class":"p-heading-icon__img"}
+              ) | safe
+            }}
+            <h4 class="p-heading-icon__title">Latest news on AI/ML</h4>
+          </div>
+          <div id="ai-articles" class="row p-divider">
+            <div><i class="p-icon--spinner u-animation--spin">Loading...</i></div>
+          </div>
+        </div>
+      </div>
+      <div class="col-4 p-divider__block">
+        <div class="p-heading-icon--muted">
+          <div class="p-heading-icon__header">
+            {{ image (
+              url="https://assets.ubuntu.com/v1/b061c401-White+paper.svg",
+              alt="File icon",
+              width="32",
+              height="28",
+              hi_def=True,
+              loading="auto|lazy",
+              attrs={"class":"p-heading-icon__img"}
+              ) | safe
+            }}
+            <h4 class="p-heading-icon__title">DOWNLOAD</h4>
+          </div>
+          <ul>
+            <li><a href="https://ubuntu.com/engage/starting-with-ai">Whitepaper: Get started with AI&nbsp;&rsaquo;</a></li>
+            <li><a href="https://drive.google.com/file/d/1it6EznImfYoV1fflc-ze8-3MhOce6wm8/view">Kubeflow Reference Architecture&nbsp;&rsaquo;</a></li>
```

Comment:
Double check this with marketing/Sam, I think this should be an asset link not a google drive link

Reply:
Changed to an assets link!